### DISCUSSION TITLE
Do we need Telegram-channel for communication and knowledge sharing?

### DISCUSSION THREAD

**Discussion by Marusyk** [Marusyk/MSA.BuildingBlocks#19] [Q&A] [answered]
Hi guys,
@kanilsz, @IgorFlyunt1 

Do you think we need a Telegram channel for better communication and maybe knowledge sharing?
We can share some interesting articles there or whatever you want
of course if you are interested in learning and have something to share

**Comment by kanilsz** [comment]
Totally agree. Maybe it will also increase the development speed (for me it is 100%) :)
My telegram: https://t.me/kanils

**Reply by Marusyk** [reply]
actually we don't need speed, just to have a communication way 😉

**Comment by IgorFlyunt1** [comment]
This is a great idea, I'm sure that will be useful for us 
My telegram: https://t.me/Igor_Flyunt

**Comment by Marusyk** [comment, marked as answer]
Cool
